blueprints/unifi/install.sh

```diff
@@ -63,10 +63,7 @@ iocage exec "${1}" sysrc unifi_enable=YES
 iocage exec "${1}" service unifi start
 
 # shellcheck disable=SC2154
-if [[ ! "${!POLLER}" ]]; then
-  echo "Installation complete!"
-  echo "Unifi Controller is accessible at https://${JAIL_IP}:8443."
-else
+if [ "${!POLLER}" = true ]; then
```

Comment:
I think we should use string comparison here:
```suggestion
if [ "${!POLLER}" == true ]; then	
```